src/stream/src/from_proto/hash_agg.rs

```diff
@@ -90,6 +92,10 @@ impl ExecutorBuilder for HashAggExecutorBuilder {
             .map(|key| *key as usize)
             .collect::<Vec<_>>();
         let [input]: [_; 1] = params.input.try_into().unwrap();
+        let mut group_key_invert_idx = vec![None; input.info().schema.len()];
+        for (group_key_seq, group_key_idx) in group_key_indices.iter().enumerate() {
+            group_key_invert_idx[*group_key_idx] = Some(group_key_seq);
+        }
```

Comment:
Should we calculate the reverted index in the builder or in the `new` function directly? 🤔

Reply:
> Should we calculate the reverted index in the builder or in the `new` function directly? 🤔

I did calculate in `new` and moved it here by other's suggestion.
Let me move back.